templates/web/search.ftl

```diff
@@ -22,66 +22,41 @@
-
-    <script src="/studio/static-assets/scripts/crafter.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-    <script src="/studio/static-assets/scripts/animator.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-    <link rel="stylesheet" type="text/css" href="/studio/static-assets/yui/assets/skins/sam/calendar.css?version=${UIBuildId!.now?string('Mddyyyy')}"/>
-    <link rel="stylesheet" type="text/css" href="/studio/static-assets/styles/search.css?version=${UIBuildId!.now?string('Mddyyyy')}"/>
-    <script src="/studio/static-assets/libs/js-cache/cache.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-    <script src="/studio/static-assets/libs/amplify/lib/amplify.core.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-    <script src="/studio/static-assets/libs/ace/ace.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-
-    <script type="text/javascript" src="/studio/static-assets/components/cstudio-search/search.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
-
-    <script>
-        CMgs = CStudioAuthoring.Messages;
-        langBundle = CMgs.getBundle("search", CStudioAuthoringContext.lang);
-        formsLangBundle = CMgs.getBundle("forms", CStudioAuthoringContext.lang);
-        siteDropdownLangBundle = CMgs.getBundle("siteDropdown", CStudioAuthoringContext.lang);
-    </script>
-
-    <script>window.entitlementValidator = '${applicationContext.get("crafter.entitlementValidator").getDescription()}';</script>
 </head>
 
 <body>
-<section class="cstudio-search"></section>
-
-<div id="cstudio-command-controls-container"></div>
-
-<script id="hb-command-controls" type="text/x-handlebars-template">
-  <div id="cstudio-command-controls">
-    <div id="submission-controls" class="cstudio-form-controls-button-container">
-      <input
-              id="formSaveButton" type="button"
-              class="cstudio-search-btn cstudio-button btn btn-primary" value="Add Selection"
-              disabled
-      >
-      <input
-              id="formCancelButton" type="button"
-              class="cstudio-search-btn cstudio-button btn btn-default" value="Cancel"
-      >
-    </div>
-  </div>
-</script>
-
-<script type="text/javascript">
-  CStudioSearch.init();
-</script>
+  <#if RequestParameters.mode?exists && mode == 'select'>
+    <#assign embedded = true />
+  </#if>
+  <div id="root" style="height: calc(100vh - 65px)"></div>
+
+  <script>
+    const opener = window.opener ? window.opener : parent.iframeOpener;
+    const searchId = CStudioAuthoring.Utils.getQueryVariable(document.location.search, 'searchId');
+    const openerChildSearchMgr = opener.CStudioAuthoring.ChildSearchManager;
+    const searchConfig = openerChildSearchMgr.searches[searchId];
+    const callback = searchConfig.saveCallback;
+    const closeSearch = () => {
+      window.close();
+      $(window.frameElement.parentElement)
+        .closest('.studio-ice-dialog')
+        .parent()
+        .remove();
+    }
```

Comment:
@rart there's still this piece of code to retrieve the callback for the `onAcceptSelection` prop. Is this ok, or do we need something extra to handle this?